File: v1/src/main/java/com/api/v1/dtos/customer/CustomerResponseDto.java
package com.api.v1.dtos.customer;

import java.time.LocalDate;

public record CustomerResponseDto(
        String fullName,
        String ssn,
        LocalDate birthDate,
        String email,
        String address,
        String phoneNumber,
        String createdAt,
        String updatedAt
) {
}

File: v2/src/main/kotlin/com/api/v2/CustomerResponseDto.kt
package com.api.v2

import java.time.LocalDate

data class CustomerResponseDto(
    val fullName: String,
    val ssn: String,
    val birthDate: LocalDate,
    val phoneNumber: String,
    val address: String,
    val email: String,
    val createdAt: String,
    val updatedAt: String?
)
